lxd/device/unix_hotplug.go

```diff
@@ -230,6 +232,7 @@ func (d *unixHotplug) loadUnixDevice() *udev.Device {
 	}
 
 	devices, _ := e.Devices()
+	var matchingDevices []*udev.Device
 	var device *udev.Device
```

Comment:
I wonder why device var here is outside of the for loop, it should be just:

```go
device := devices[i]
```

Reply:
Agreed